### ISSUE TITLE
List of names of employees for onrr.gov 508 compliance project

### ISSUE THREAD

**Issue by Maroyafaied** [ONRR/onrr.gov-site#1474]

**Comment by cthomasONRR** [comment]
After going though issues and emails, this is what I have:

Jodie Peterson, Antoinette Urisote, Christina Mathers, Carrie Garcia, Carrie Anderson, Lori Yokomizo, Shannon Hahn, Yvette Smith, Andrea Scharton, Evelyn Shropshire, Bill Blackburn, Luis Aguilar, Alexis Long, Noelle White, Kerri Gallant, Jessica Bowlen.

**Comment by cthomasONRR** [comment]
There are probably more people who were remediating, but these are the ones who emailed me with docs to update or remove.

**Comment by mcharg** [comment]
We probably have more listed in the document updater column in the content audit sheet within the 508 sheet.

**Comment by cthomasONRR** [comment]
@mcharg I checked and that column is blank in the Content Audit tab. The Accessibility Tracking tab only tracks the GS15 owner.

**Comment by Maroyafaied** [comment]
A few other names are 
Chris Mentasti
Aaron Lindquist
Candace Merrell
@mcharg can you check from the 508 participant list if there are any document updaters we interviewed that need to be included.

**Comment by mcharg** [comment]
@Maroyafaied I'll take a look tomorrow and also see if I can recover the spreadsheet to a point where the column wasn't blank.

**Comment by Maroyafaied** [comment]
@mcharg @cthomasONRR I got in the spreadsheet and the column was still there in one of the tabs. Here's the list I gave Nichelle, if you think of anyone else I can email their name. 

 Employees updating documents: Christina Mathers, Jodie Peterson, Antoinette Urisote, Carrie Garcia, Carrie Anderson, Lori Yokomizo, Shannon Hahn, Yvette Smith, Andrea Scharton, Evelyn Shropshire, Bill Blackburn, Luis Aguilar, Alexis Long, Noelle White, Kerri Gallant, Jessica Bowlen, Jessica Manko, Samantha McFarland, Elise Athens, Walter Weathersby, Candace Merrell, Aaron Lindquist, Chris Mentasti.

**Comment by mcharg** [comment]
Additional people copied on the RRM email when they finished the first time.

Jon Swedin, Lisa Reimer, Jill Heinen, John Erle

Do we also want to include our cross-trainers who helped?

Looks like you got the participants from the 508 interviews.

Additional participants from the content process interviews who update documents:
Damion Schulte, Hannah Bade, Janice Lopez, Sandra Allen, Amanda Garrison, Cheri Baker

Additional People on the ONRR-wide 508 team who may have also updated documents:
Cassady Blakesley (definitely did some audit docs, but maybe those are internal and is updating the style guide), Jesse Kissler, Tim Martin, Cheryl Howley, Jennifer Wells

**Comment by mcharg** [comment]
There's also a tab with content updaters in the market map. https://doimspp.sharepoint.com/:x:/r/sites/NRRD/Shared%20Documents/General/ONRR.gov/User%20Research/ONRR.gov%20market%20map.xlsx?d=wc989d84457f740908c35de6f819f178e&csf=1&web=1&e=dYVKBu

Names that I don't think are listed above (a lot of these update the contacts lists or pricing): Jessica Penvose, Cory Forgrave, Jessica Sheets, Sean Boice, Jared Good, Jennifer Dougherty, Matthew Moore, Crystel Sandos, Matt Collins, Kim Werner, Christine Madigan (says she updated 508 docs for Indian), Heidi Badaracco, Karl Wunderlich, Michael Marchetti

**Comment by Maroyafaied** [comment]
@mcharg dod all those people specifically worked on the 508 project for onrr.gov? I know some supervisors didn't remediate documents but were included in emails like John Swedin, Karl W, Heidi B, Michael Marchetti and Matti Collins. And yes we should include the cross trainers.

**Comment by mcharg** [comment]
I'm not sure.   I know Kim Werner, Amanda Garrison, Lisa Reimer, Jill Heinen, Janice Lopez, Cheri Baker, Jessica Sheets, Christine Madigan for sure.
